readme.md

```diff
@@ -210,44 +240,51 @@ To do so:
+  - Create a Remote Repo called merge-conflicts and add this repo locally as a remote repo for your merge-conflicts directory.
+  - Make sure to save and commit local changes and push up to the Remote Repo
+  - Add the Secondary student as a Collaborator (search github for how to do this)
 
-Merge conflicts:
+  Secondary Students Instructions:
+  - After they are added as a Collaborator, they should clone the same repo. Do not fork the Repo.
 
-1. Both students make changes on your respective working dirs, to the same line, of the same file.  Commit and push the changes.
-- One collaborator will be forced to solve the merge conflicts, when they pull the changes to the same line.
-- Practice resolving the commit.
-- Repeat, ensuring the other partner pushes second, forcing them to resolve the new conflict.
+3. Both the Primary and Secondary should make changes locally on the same "master" branch
+  - Modify the index.html
+  - Add and Commit Changes Locally.
 
-## Homework
-
-From this point on, all homework submissions should be a pull request from a feature (or 'topic') branch, named `<your_name>_solution`.
+5. Merging commits:
+  - The Primary Student should push up their changes first
+  - Then, the Secondary Student should do the same and try pushing up their changes
 
+6. Merge conflicts:
+  - When the Secondary Student Tries to Push their commits, there should be merge conflicts
+  - The Secondary Student should work locally (with the Primary) to resolving the merge conflicts.
+  - Once completed, push up changes to the remote repo
 
-## `git mergetool` (An exercise for the reader)
+7. Pulling Changes:
+  - Now, the Primary student should pull down the changes from the remote repo and work to resolve the      merge conflicts
 
-Some merge conflicts can be quite confusing.  Git provides a way to use a visual tool to assist in resolving the merge conflicts.
+## Closing (5 min)
 
-If you have installed XCode, an decent tool (opendiff) is used.  For this class, we will use [KDiff3](http://kdiff3.sourceforge.net/).  Here's an [example, with images](http://www.gitguys.com/topics/merging-with-a-gui#Merging_with_kdiff3), including a way to try it first.
+Review Learning Objectives:
+* Explain what a branch is in git
+* Create, merge and delete branches on local and remote repositories
+* Describe how branching and merging allows for collaboration during development
+* Resolve a merge conflict
 
-- Install via homebrew: `brew install kdiff3`
-- Setup KDiff3 as your merge too using [these instructions](http://naleid.com/blog/2012/01/12/how-to-use-kdiff3-as-a-3-way-merge-tool-with-mercurial-git-and-tower-app).  
-  - Be sure to skip the "Installation" steps.
-  - Stop at "Git Tower Integration"
-- When you have a merge conflict, you can type `git mergetool` to see a 3-way merge and the merge results.  Like this: ![](http://naleid.com/images/2012/01/kdiff3_merge_window_fixed.png)
+### Homework
```

Comment:
feel like a screencast demoing this process would be very helpful - I can make it for you if you like and/or we can pair on it